site/content/introduction/01-introduction.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Introduction"
+weight: 10
+---
+
+# `eksctl` - a CLI for Amazon EKS
```

Comment:
The top part of it seems to be the same as the home page... something that we will need to address.

Reply:
yes, I'm not sure what do do between those two. What do you think?